src/sections/CareersHero/index.css

```diff
@@ -0,0 +1,78 @@
+@import "../../styles/all.css";
+
+.root {
+  padding-bottom: $spacing-224;
+
+  @include Breakpoint-tabletOnly {
+    padding-bottom: $spacing-136;
+  }
+
+  @include Breakpoint-mobileOnly {
+    padding-bottom: $spacing-136;
+  }
+}
+
+.title-desktop {
+  display: flex;
+  flex-direction: column;
+
+  margin-top: calc($spacing-224 - $navbar-height);
+  margin-bottom: $spacing-136;
+
+  @include Breakpoint-tabletAndBelow {
+    display: none;
```

Comment:
`title-desktop` will display on desktop and tablet, we're using desktop layout for tablet and mobile. Sorry, I was wrong in the previous comment and I had to check on other pages.